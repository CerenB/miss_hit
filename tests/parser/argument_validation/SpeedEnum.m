% From https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html

classdef SpeedEnum < int32
    enumeration
        Full   (100)
        Half   (50)
        Stop   (0)
    end
end
