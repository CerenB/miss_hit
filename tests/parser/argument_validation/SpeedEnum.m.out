----------------------------------------------------------------------
--  Parse tree for SpeedEnum.m
Root: Class_File
  Classdef: Class_Definition
    Name: Identifier <SpeedEnum>
    Superclasses: Identifier <int32>
    Enumerations: Enumeration Special_Block
      Items: Class_Enumeration
        Name: Identifier <Full>
        Arguments: Number_Literal <100>
      Items: Class_Enumeration
        Name: Identifier <Half>
        Arguments: Number_Literal <50>
      Items: Class_Enumeration
        Name: Identifier <Stop>
        Arguments: Number_Literal <0>
----------------------------------------------------------------------
In SpeedEnum.m, line 3
| classdef SpeedEnum < int32
|                    ^ info: this token is not linked to the ast
In SpeedEnum.m, line 5
|         Full   (100)
|                ^ info: this token is not linked to the ast
In SpeedEnum.m, line 5
|         Full   (100)
|                    ^ info: this token is not linked to the ast
In SpeedEnum.m, line 6
|         Half   (50)
|                ^ info: this token is not linked to the ast
In SpeedEnum.m, line 6
|         Half   (50)
|                   ^ info: this token is not linked to the ast
In SpeedEnum.m, line 7
|         Stop   (0)
|                ^ info: this token is not linked to the ast
In SpeedEnum.m, line 7
|         Stop   (0)
|                  ^ info: this token is not linked to the ast
In SpeedEnum.m, line 8
|     end
|     ^^^ info: this token is not linked to the ast
In SpeedEnum.m, line 9
| end
| ^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
